tests/test_order_ContinuousCoordination.py

```diff
@@ -0,0 +1,68 @@
+# Copyright (c) 2010-2023 The Regents of the University of Michigan
+# This file is from the freud project, released under the BSD 3-Clause License.
+
+import numpy.testing as npt
+import pytest
+
+import freud
+
+
+class TestContinuousCoordination:
+    """Test fixture for ContinuousCoordination"""
+
+    def setup_method(self):
+        """Initialize a box with randomly placed particles"""
+        box_size = 10
+        num_points = 1000
+        self.box, self.pos = freud.data.make_random_system(
+            box_size, num_points, seed=123
+        )
+        # 0.0 is the same as just counting neighbor and can be used for testing
+        self.powers = [0.0, 2.0, 4.0, 8.0]
+        self.compute_log = True
+        self.compute_exp = True
+        self.coord = freud.order.ContinuousCoordination(
+            self.powers, self.compute_log, self.compute_exp
+        )
```

Comment:
Let's add test where we give the constructor many different combinations of coordinations and make it sure it computes the right number of coordinations.